packages/browser/jest.config.js

```diff
@@ -4,4 +4,5 @@ module.exports = {
   ...baseConfig,
   testEnvironment: 'jsdom',
   testMatch: ['<rootDir>/test/unit/**/*.test.ts'],
+  prettierPath: null,
```

Comment:
For using the `toMatchInlineSnapshot` method to test. Otherwise, you should update some pkgs with the major version, I think. So it may be a better way bc didn't break anything.

BTW, Why not use `toMatchInlineSnapshot` but write the expected value manually? This method is very inefficient 🤔

Reply:
Because snapshots are affected by prettier and sometimes are harder to read, I think we prefer directly asserting on the object with `toEqual`. Can we use that instead of `toMatchInlineSnapshot`?